app/domain/authentication/authn_jwt/jwt_initial_input_validation.rb

```diff
@@ -0,0 +1,40 @@
+require 'command_class'
+
+# This class
+module Authentication
+  module AuthnJwt
+
+    JWTInitialInputValidation ||= CommandClass.new(
+      dependencies: {
+        enabled_authenticators: InstalledAuthenticators.enabled_authenticators_str(ENV),
```

Comment:
not sure that this will work. i think it needs to be an input in case the env changes.

Reply:
I'll assume it as an input parameter